src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {Client} from 'app/api';
+import {Config, Organization} from 'app/types';
+import {RouteComponentProps} from 'react-router/lib/Router';
+import AsyncView from 'app/views/asyncView';
+import withApi from 'app/utils/withApi';
+import withOrganization from 'app/utils/withOrganization';
+import withConfig from 'app/utils/withConfig';
+import withLatestContext from 'app/utils/withLatestContext';
+import {PageContent} from 'app/styles/organization';
+import {t} from 'app/locale';
+
+import Button from 'app/components/button';
+import Sidebar from 'app/components/sidebar';
+
+type RouteParams = {
+  orgId: string;
+  dataId: string;
+};
+
+type Payload = {
+  storage_url?: string;
+  expired_at?: string;
+  status?: 'EXPIRED' | 'INVALID' | 'VALID' | 'EARLY';
+};
+
+type Props = {
+  api: Client;
+  organization: Organization;
+  config: Config;
+} & RouteComponentProps<RouteParams, {}>;
+
+type State = {
+  payload: Payload;
+} & AsyncView['state'];
+
+class DataDownload extends AsyncView<Props, State> {
+  getDefaultState(): State {
+    return {
+      ...super.getDefaultState(),
+      payload: {
+        storage_url: '',
+        expired_at: '',
+      },
+    };
+  }
+
+  getTitle(): string {
+    return t('Download Center');
+  }
+
+  getEndpoints(): [string, string][] {
+    const {orgId, dataId} = this.props.params;
+    return [['payload', `/organizations/${orgId}/data-export/${dataId}/`]];
```

Comment:
@leeandher I feel like we could come up with a better name than `payload`

Reply:
switched it to `Download`